app/taran_hw20/tests/test_task.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from app.task import Task
+
+
+@pytest.mark.task_suite
+@pytest.mark.parametrize("title, description, date, priority, expected_result",
+                         [("title1", "desc1", "1111-11-11", "low",
+                           ["title1", "desc1", "1111-11-11", "low", False])])
```

Comment:
It is better to use dict instead of list here. Because names are always clearer than indexes.

```
@pytest.mark.task_suite
@pytest.mark.parametrize(
    "title, description, date, priority, expected_result",
    [("title1", "desc1", "1111-11-11", "low",
    {"title": "title1", "description": "desc1", "due_date": "1111-11-11", "priority": "low", "completed": False})])
def test_task_to_dict(title, description, date, priority, expected_result):
    task = Task(title, description, date, priority)
    assert task.title == expected_result["title"]
    assert task.description == expected_result["description"]
    assert task.due_date == expected_result["due_date"]
    assert task.priority == expected_result["priority"]
    assert task.completed == expected_result["completed"]
```